awx/main/middleware.py

```diff
@@ -64,11 +74,12 @@ def save_profile_file(self, request):
         return filepath
 
 
-class ActivityStreamMiddleware(threading.local):
+class ActivityStreamMiddleware(MiddlewareMixin, threading.local):
 
-    def __init__(self):
+    def __init__(self, get_response=None):
         self.disp_uid = None
         self.instance_ids = []
+        self.get_response = get_response
```

Comment:
In this case I think that relying on the behavior of `MiddlewareMixin` would be slightly preferable, using a `super()` call.

Reply:
Done!